### PR TITLE
Add note about Kodi being on for import to work

### PR DESCRIPTION
**Pull request by OnFreund**
## Proposed change

Add a note to the Kodi documentation to make sure it's on when importing configuration.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: https://github.com/home-assistant/core/issues/40112

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by OnFreund** [comment]
I think this note should also be added to the "breaking changes" section of the release notes.

File: source/_integrations/kodi.markdown [hunk 1]
```diff
@@ -30,6 +30,7 @@ If you previously had Kodi configured through `configuration.yaml`, it's advisab
 If you do not remove it, your configuration will be imported with the following limitations:
 * Your turn on/off actions will not be imported. This functionality is now available through device triggers.
 * You may have duplicate entities.
+* Kodi must be on when HA is loading for the first time for the configuration to be imported.
```
Comment (frenck) [inline source/_integrations/kodi.markdown]:
```suggestion
* Kodi must be on when Home Assistant is loading for the first time for the configuration to be imported.
```

**Comment by frenck** [comment]
@OnFreund Please add the note to that specific parent PR or open up a PR against the release blog

**Comment by OnFreund** [comment]
Thanks @frenck . I edited the breaking changes note in the original kodi integration PR (https://github.com/home-assistant/core/pull/38551). Is that enough?

**Comment by frenck** [comment]
@OnFreund No, as it already was marked before and thus has been processed. Please PR against the release notes on the RC branch for that.

**Review by frenck — APPROVED** [review]

**Comment by OnFreund** [comment]
Ok, here it is:
https://github.com/home-assistant/home-assistant.io/pull/14512

**Comment by fermulator** [comment]
hey all not sure I am following - a later version has broken the resiliency of startup if a kodi device isn't turned on, and we're only documenting it? is there a ticket to actually fix the issue?

**Comment by OnFreund** [comment]
@fermulator sounds like you might mean this: https://github.com/home-assistant/core/issues/40450, which is a different issue

**Pull request merged**